package.json

```diff
@@ -54,6 +54,7 @@
     "bech32": "^2.0.0",
     "bip174": "^2.1.1",
     "bs58check": "^3.0.1",
+    "typedoc-plugin-bitcoinjs-runcase": "^1.0.1",
```

Comment:
Shouldn't this be a devDependency?

Reply:
Yes, it should me. I remember this **but maybe something is wrong!!**

I want to tell you about another thing. 
**I added an executable environment to the document to help us more flexibly display related cases and provide user testing**. 
The background is as follows

1. We have a lot of test cases. Moving too many cases will affect the organizational structure of our code, and the workload will be huge.
2. For related cases, single testing can be combined with the context to understand more, more comprehensively and more efficiently.

To achieve this goal, I did the following things, which also cost me a lot of time:

1. Build bitcoinjs-lib browser version
2. Develop typedoc-plugin-bitcoinjs-runcase plug-in to support sneaking into the executable environment
3. Modify relevant comment tags

You can view the relevant effects here: https://jasonandjay.github.io/bitcoinjs-lib/functions/address.fromBase58Check.html

![1](https://github.com/bitcoinjs/bitcoinjs-lib/assets/8192412/9ee5dec8-0ced-47c9-8c24-68302f0f0f59)
![2](https://github.com/bitcoinjs/bitcoinjs-lib/assets/8192412/3c1f3176-58de-4dd4-b619-147805599c5e)

I think this is a great feature, but I need your help evaluating whether it's suitable

Reply:
That way we can tell the user to test here and find more cases in the single test

Then we only need to **give a correct usage method in example** (I customized a @case tag), so that the construction efficiency of our document will be much higher

Reply:
@junderw please take a look

Reply:
@jasonandjay  `typedoc-plugin-bitcoinjs-runcase` is a production dependency and has `react` and `react-dom` as dependencies. That doesn't belong in bitcoinjs at all.

Reply:
@pajasevi yes, i will change it later, i want to know is this the right direction to add an executable environment to the document to simplify the case in the single test? so as to proceed with the next step.

Reply:
This will take a while to review.

Reply:
Perhaps this should be in a follow up pr

Reply:
Review typedoc plugin or comments in our repository?

Reply:
Adding `typedoc-plugin-bitcoinjs-runcase` should be in a separate PR is what I'm saying.